util.c

```diff
@@ -94,7 +94,7 @@ unsigned attr_as_unsigned(xmlNode *node, const char *attr, int *errors)
 	return (unsigned int) strtoul((char*)value, NULL, 10);
 }
 
-const char *attr_as_string(xmlNode *node, const char *attr, int *errors)
+xmlChar *attr_as_string(xmlNode *node, const char *attr, int *errors)
```

Comment:
this function converts a xml attribute into a string. i don't understand why you want to use xmlChar, since the purpose is to extract information from xml.

Reply:
It's better because XML API returns xmlChar which is unsigned char. As we use libxml2 there is no reason to use normal strcmp. With this change things becomes more clear for use.

Reply:
I have updated patch with commit message with all reasons.